src/Atmos/Parameterizations/CloudPhysics/Activation.jl

```diff
@@ -0,0 +1,154 @@
+
+
+Pkg.add("SpecialFunctions")
+module Activation
+
+using Thermodynamics
+using CLIMAParameters
+using CLIMAParameters.Atmos.Microphysics_0M
+const APS = AbstractParameterSet
+export activation_cloud_droplet_number
+# TODO: Check over these ^^^^
+
+"""
+    Isabella Dula and Shevali Kadakia
+    
+    This file has all the functions for the parametrization, according to the model
+    given in Abdul-Razzak and Ghan (2000).
+    
+"""
+#--------------------------------------------------------------------#
+# TODO: 
+# --Write function to determine mass fraction of soluble 
+#   material (epsilon)
+#--Determine P_saturation origins (do we get this from model?)
+#--Double check math syntax
+#--------------------------------------------------------------------#
+# CONSTANTS 
+
+WTR_MM = 0.01801528 # kg/mol
```

Comment:
As a first TODO let's get rid of those constants and start using the CLIMAParameters constanst